Orleans/src/Microsoft.Orleans.Host.Abstractions/Microsoft.Orleans.Host.Abstractions.csproj

```diff
@@ -52,9 +52,12 @@
     <Reference Include="System.Xml" />
   </ItemGroup>
   <ItemGroup>
+    <Compile Include="AzureStorageProvider.cs" />
```

Comment:
definitely don't belong here, these are concrete classes, not part of the abstraction.